### DISCUSSION TITLE
PDFCreator Free Package Takeover

### DISCUSSION THREAD

**Discussion by karlo-mirkovic** [chocolatey-community/chocolatey-packages#2447] [General]
As a result of recent initiatives Avanquest pdfforge, the developers of the PDFCreator software, would like to publish and maintain all PDFCreator editions on chocolatey. While the packages of the other editions are awaiting approval, we noticed that PDFCreator Free has already been published (with automatic update) to the chocolatey community repository. We (Avanquest pdfforge) would like to take over the maintenance of that package.

Since contacting the current maintiners of the package is not possible, what would be the necessary steps in order to achieve this?

**Comment by pauby** [comment]
We have no issues picking the package up from the Chocolatey Community and updating it directly. Could you confirm the other packages that you are maintaining?

All we need here is confirmation that the package is ready to submit. Once that is done, you would need to:

1. Submit a PR to this repository, removing it.
2. Confirm your Chocolatey username.

**Reply by karlo-mirkovic** [reply]
I am currently the maintainer of 3 other PDFCreator editions whose packages are currently in a **Waiting for review** state.
The packages are the following: [PDFCreator Professional](https://community.chocolatey.org/packages/pdfcreator-professional/5.2.1), [PDFCreator Terminal Server](https://community.chocolatey.org/packages/pdfcreator-terminal-server/5.2.1), and [PDFCreator Server](https://community.chocolatey.org/packages/pdfcreator-server/3.2.0)
[Here is a link to the PR](https://github.com/chocolatey-community/chocolatey-packages/pull/2464), removing PDFCreator from the community repository.
[Here is my profile](https://community.chocolatey.org/profiles/karlo.wav) on community.chocolatey.org.

**Reply by karlo-mirkovic** [reply]
I hereby confirm that the package is ready to submit. Waiting on the PR.

**Reply by pauby** [reply]
Thanks for confirming the other packages. 

We have been chatting internally, and we are unable to determine a way to easily verify you are from Avanquest. As I cannot find an email address on the [website](https://www.pdfforge.org/), I assume that the email domain @pdfforge.org is correct.

Assuming your account on the Chocolatey Community Repository is using your @pdfforg.org email address, can you go to the [package page](https://community.chocolatey.org/packages/pdfcreator), ensure you are logged in with your account, click 'Contact Site Admins' and send a message linking to this thread?

**Reply by karlo-mirkovic** [reply]
I'd gladly verify that I'm from Avanquest.
As pdfforge GmbH has become Avanquest pdfforge GmbH, our email domains have moved from @pdfforge.org to @avanquest.com. My account on the Chocolatey Community Repository is using my @avaquest.com email address.
I have contacted you from that account.

**Reply by pauby** [reply]
Thanks for con firming that. I confirm I received the email from the @avanquest.com domain.

**Comment by JPRuskin** [comment]
The package is now added to your user, the PR has been merged, and we've removed the package from `chocolatey-community`.

I think we can close this.

Thank you for taking the package over!
